notebooks/en/_toctree.yml

```
@@ -94,6 +94,8 @@
           title: Fine-Tuning a Vision Language Model (Qwen2-VL-7B) with the Hugging Face Ecosystem (TRL)
         - local: multimodal_rag_using_document_retrieval_and_reranker_and_vlms
           title: Multimodal RAG with ColQwen2, Reranker, and Quantized VLMs on Consumer GPUs
+        - local: multimodal_rag_using_document_retrieval_and_smol_vlm
+          title: Smol Multimodal RAG: Building with ColSmolVLM and SmolVLM on Colab's Free-Tier GPU
```

Can't have a `:` in the title

```suggestion
          title: Smol Multimodal RAG, Building with ColSmolVLM and SmolVLM on Colab's Free-Tier GPU
```